service/synapse-service-rest/src/main/java/io/americanexpress/synapse/service/rest/controller/reactive/BaseReactiveDeleteMonoController.java

```diff
@@ -0,0 +1,30 @@
+package io.americanexpress.synapse.service.rest.controller.reactive;
+
+import io.americanexpress.synapse.service.rest.controller.BaseController;
+import io.americanexpress.synapse.service.rest.service.BaseDeleteReactiveService;
+import io.swagger.v3.oas.annotations.Operation;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import reactor.core.publisher.Mono;
+
+public abstract class BaseReactiveDeleteMonoController<S extends BaseDeleteReactiveService> extends BaseController<S> {
+
+    @DeleteMapping("/{identifier}")
+    @ResponseStatus(HttpStatus.NO_CONTENT)
+    @Operation(tags = "Delete Operation", summary = "Deletes a resource reactively")
+    public Mono<ResponseEntity<Void>> delete(@PathVariable String identifier) {
+        logger.entry(identifier);
+
+        var serviceResults = service.delete(identifier);
+        var responseEntity = serviceResults
+                .map(ResponseEntity::ok)
+                .defaultIfEmpty(ResponseEntity.noContent().build());
```

Comment:
For this one i think it should return NoContent (204) if successful instead of OK (200)

Reply:
im thinking 404?